docs/modules/operation/nav.adoc

```diff
@@ -129,7 +138,7 @@
 ** xref:elasticsearch/features/alarm-history.adoc[]
 
 * xref:flows/introduction.adoc[]
-** xref:flows/basic.adoc[]
+** xref:flows/setup.adoc[]
```

Comment:
@Bonrob2 Does this conflict with your flow changes, or are your updates still pending?